ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js

```diff
@@ -939,10 +971,12 @@ export default {
         firstWeave: {
           en: 'Go North (donut bottom/circle top)',
           de: 'Geh nach Norden (Donut unten/Kreise oben)',
+          fr: 'Allez au nord (donut bas/cercle haut)',
         },
         secondWeave: {
           en: 'Stay South (square bottom/circle top)',
           de: 'Geh nach Süden (Viereck unten/Kreise oben)',
+          fr: 'Restez au sud (fond carré/cercle haut)'
```

Comment:
Maybe it's confused, but I think this is the missing comma.

Reply:
I have however checked several times. Thank you and sorry for the unnecessary inconvenience.
Just a quick note, all local **npm run tests were successful**. I am troubled that I had all of these not found errors. Is the command `npm run test` still?